tools/configure.py

```diff
@@ -475,7 +475,8 @@ def default_from_script_path(optname, orig, alternatives):
         'duk_internal.h',
         'duk_jmpbuf.h',
         'duk_exception.h',
-        'duk_js_bytecode.h',
+        'duk_js_arith.c',
+		'duk_js_bytecode.h',
```

Comment:
Whoops, notepad++ usually preserves the proper whitespace when adding lines, not sure what happened here.  I'll fix it when I rebase in a bit.